test/passing/tests/js_source.ml

```diff
@@ -7603,3 +7603,15 @@ let x =
     ~f:(fun thing ->
       something that reaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaally needs wrapping)
 ;;
+
+let x =
+  foo [ A; B ] ~f:(fun thing ->
```

Comment:
This is not testing the new feature. I think this formatting is due to `is_simple`, so it was the same before this PR.

Reply:
Yes I've just put these 2 let-bindings next to each other so we can quickly check in the .ref if they are formatted the same.